### DISCUSSION TITLE
TimeSeriesPredictor redirect verbose output to file

### DISCUSSION THREAD

**Discussion by lucical** [autogluon/autogluon#3765] [Q&A]
Hi,

How can I save the verbose output from the terminal when I train the `TimeSeriesPredictor` to a file? I found [these answers](https://github.com/autogluon/autogluon/issues/2325) for `TabularPredictor` but none of it works for Time Series prediction and there is no support for logging implemented like there `predictor = TabularPredictor(..., log_to_file=True)`.

Thanks!

**Comment by canerturkmen** [comment]
Hi @lucical 👋 ,

Yes, you are right that `TimeSeriesPredictor` does not support this argument. It's now in the backlog to add it. In the meanwhile, may I suggest the following workaround:

```
from autogluon.common.utils.log_utils import add_log_to_file
from autogluon.timeseries.predictor import logger as LG
from autogluon.timeseries import TimeSeriesDataFrame, TimeSeriesPredictor

add_log_to_file("./my.log", LG)

if __name__ == "__main__":
    
    df = TimeSeriesDataFrame("https://autogluon.s3.amazonaws.com/datasets/timeseries/m4_hourly_subset/train.csv")
    pred = TimeSeriesPredictor()
    
    pred.fit(
        df,
        hyperparameters={
            "DeepAR": {
                "max_epochs": 1,
                "num_batches_per_epoch": 1,
            },
        },
    )
    
    LG.info(
        pred.leaderboard()
    )

```
